src/index.html

```diff
@@ -6,12 +6,168 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label>Surname:
+          <input
+            type="text"
+            name="surName"
+            autocomplete="off"
+          >
+        </label>
+        <br>
+        <label>Name:
+          <input
+            type="text"
+            name="firstName"
+            autocomplete="off"
+          >
+        </label>
+        <br>
+        <label>How old are You?
```

Comment:
1) it's not a good practice to use `<br />` to make distance between elements. Use styles 
2) add an extra line between `multiline` siblings
```suggestion
        </label>

        <label>How old are You?
```